new_file.txt

```diff
@@ -1,3 +1,5 @@
 This is random text
 Adding a new line to this
-Added a new line again!
\ No newline at end of file
+Added a new line again!
+
+Let there be chaos!
```

Comment:
@vishishtpriyadarshi you are not good for society.

Reply:
[Embrace the chaos](https://www.youtube.com/watch?v=4Mp_AY0eflM) 😛